### PR TITLE
Placeholder main script for model retraining on negative samples; glo…

### PR DESCRIPTION
**Pull request by pastorep**
…bal refactoring

### CODE REVIEW

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (prakruti) [inline InferenceSystem/src/model_retrain_negative_samples.py:1]: Errant newline?

File: InferenceSystem/src/globals.py
Comment (Molkree) [reply InferenceSystem/src/globals.py:9]: Also no newline here

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (Molkree) [inline InferenceSystem/src/model_retrain_negative_samples.py:46]: Also no newline here

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (Molkree) [inline InferenceSystem/src/model_retrain_negative_samples.py:8]:
```suggestion
    Example usage:
```

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (Molkree) [inline InferenceSystem/src/model_retrain_negative_samples.py:17]: Backslashes are not needed here. I'm wondering, is it some formatter that puts them here or were they put in manually? 🤔
If it's the latter, I suggest using Black formatter. For example this is how it formats these statements:
```py
    parser.add_argument(
        "hydrophone",
        choices=list(globals.S3_STREAM_URLS.keys()),
        required=True,
        help="Hydrophone location",
    )
```

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (Molkree) [inline InferenceSystem/src/model_retrain_negative_samples.py:22]: In example usage and argument help lines you omit seconds (`"2020-07-25 19:15"`) but here you use seconds, this is going to fail at runtime. I suggest adding seconds to examples rather than slashing accuracy to minutes only.

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (Molkree) [inline InferenceSystem/src/model_retrain_negative_samples.py:27]: Argument is called `hydrophone` above, should be synced.
Additionally I'm not sure it has to be asserted, `ArgumentParser` will do it for us because it is just `choices`.

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (Molkree) [inline InferenceSystem/src/model_retrain_negative_samples.py:9]: Huh, what about this `dataset_folder` argument?

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (Molkree) [inline InferenceSystem/src/model_retrain_negative_samples.py:40]:
```suggestion

```

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (Molkree) [inline InferenceSystem/src/model_retrain_negative_samples.py:43]:
```suggestion
    # Test new trained model with current evaluation benchmark and Output graphs and save all to auto-PR
```

File: InferenceSystem/src/globals.py
Comment (Molkree) [reply InferenceSystem/src/globals.py:9]: I don't know actually! 😅 (but I guess there's no guide)
Newlines at end of files should be in all style guides. But for everything else, I'd recommend to just use Black. It's now the default formatter in VSCode at least and lots of open source projects use it.

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (prakruti) [reply InferenceSystem/src/model_retrain_negative_samples.py:9]: dataset_folder does not need to be parameter.

File: InferenceSystem/src/globals.py
Comment (pastorep) [reply InferenceSystem/src/globals.py]: @Molkree thanks for the heads up! I switched formatter to Black from default PEP8, so my code should now be more in line with the rest of the repo :)

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (pastorep) [reply InferenceSystem/src/model_retrain_negative_samples.py]: addressed

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (pastorep) [reply InferenceSystem/src/model_retrain_negative_samples.py]: addressed

**Comment by prakruti** [comment]
@Molkree please could you take a second look and approve if things look good? @pastorep please squash and merge after getting approval from Dmitry. Thanks!

**Comment by prakruti** [comment]
@pastorep looks like @Molkree might be unavailable to do a second review at the moment. I can still merge it if you think it's good to go. Let me know, thanks!

File: InferenceSystem/src/model_retrain_negative_samples.py
Comment (Molkree) [inline InferenceSystem/src/model_retrain_negative_samples.py:24]:
```suggestion
    td = globals.MAX_NEGATIVE_SAMPLE_RETRAIN_DATETIME_DELTA
    max_timedelta_string = (
        f"{td.seconds // 3600}H:{(td.seconds // 60) % 60}M{td.seconds % 60}S"
    )
    parser.add_argument(
        "--end_time",
        type=str,
        required=True,
        help=f"End time in PST in following format 2020-07-25 20:15:00. Cannot be greater than (start_time + {max_timedelta_string})",
    )
```
Might want to get the minutes from the global instead of hardcoding

**Review by Molkree — APPROVED** [review]
Just one tiny suggestion, other than that looks good!

**Pull request merged**
